clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesDaemonSetHandler.java

```diff
@@ -91,9 +91,8 @@ protected KubernetesCachingAgentFactory cachingAgentFactory() {
 
   @Override
   public Status status(KubernetesManifest manifest) {
-    V1beta2DaemonSet v1beta2DaemonSet =
-        KubernetesCacheDataConverter.getResource(manifest, V1beta2DaemonSet.class);
-    return status(v1beta2DaemonSet);
+    V1DaemonSet v1DaemonSet = KubernetesCacheDataConverter.getResource(manifest, V1DaemonSet.class);
```

Comment:
Just to be consistent with the other handlers, I think we can add the collection of `SUPPORTED_API_VERSIONS` with only `apps/v1` here, and throwing the `UnssuportedVersionException` if something else is passed in.